qpc/report/deployments.py

```diff
@@ -9,7 +9,7 @@
 # along with this software; if not, see
 # https://www.gnu.org/licenses/gpl-3.0.txt.
 #
-"""ReportSummaryCommand is used to show summary report information."""
+"""ReportDeploymentsCommand is used to show summary report information."""
```

Comment:
change this docstring to reflect deployments report information

Reply:
👍